spec/features/variables/delete_variable_spec.rb

```diff
@@ -41,7 +39,7 @@
 
         it 'informs the user of the number of collection associations that will also be deleted' do
           # 2 associated collections
-          expect(page).to have_content('This variable is associated with 2 collections. Deleting this variable will also delete the collection associations.')
+          expect(page).to have_content('This variable is associated with 1 collections. Deleting this variable will also delete the collection associations.')
```

Comment:
Not a big deal but 'collections' should be singular 'collection'

Reply:
While I agree that the English is wrong, my goal with this ticket is to get master building.  The change in CMR should have a matching ticket in MMT to change our interface and our features, because a fair chunk more than the English is now obsolete.